src/main/java/com/example/ecom/controller/CityController.java

```diff
@@ -0,0 +1,67 @@
+package com.example.ecom.controller;
+
+import com.example.ecom.model.Area;
+import com.example.ecom.model.City;
+import com.example.ecom.repository.AreaRepository;
+import com.example.ecom.repository.CityRepository;
+import org.springframework.web.bind.annotation.*;
+
+import javax.persistence.TypedQuery;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/city")
+public class CityController {
+
+    CityRepository cityRepository;
+    AreaRepository areaRepository;
+
+    public CityController(CityRepository cityRepository, AreaRepository areaRepository){
+        this.areaRepository = areaRepository;
+        this.cityRepository = cityRepository;
+    }
+
+    // Get all cities
```

Comment:
You should write a better documentation Get all cities is not good, it is same as the method name it is useless. remained me tomorrow to show you how to right it.